From 8d565a7d235ab2d4b736127d9789987ed20efcce Mon Sep 17 00:00:00 2001
From: John Clark <inindev@gmail.com>
Date: Sat, 29 Jul 2023 02:41:39 +0000
Subject: [PATCH 1/2] add sfc alias

Signed-off-by: John Clark <inindev@gmail.com>
---
 arch/arm/dts/rk356x-u-boot.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/dts/rk356x-u-boot.dtsi b/arch/arm/dts/rk356x-u-boot.dtsi
index c340c2bba6..e7b46bb2d4 100644
--- a/arch/arm/dts/rk356x-u-boot.dtsi
+++ b/arch/arm/dts/rk356x-u-boot.dtsi
@@ -9,6 +9,7 @@
 	aliases {
 		mmc0 = &sdhci;
 		mmc1 = &sdmmc0;
+		spi4 = &sfc;
 	};
 
 	chosen {
-- 
2.40.1

