From 76badff1da1c4c3f9187b24d5f129b3a96b5e08d Mon Sep 17 00:00:00 2001
From: John Clark <inindev@gmail.com>
Date: Sun, 21 May 2023 09:33:22 -0400
Subject: [PATCH 15/16] round-up on spi read size

Signed-off-by: John Clark <inindev@gmail.com>
---
 common/spl/spl_fit.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/common/spl/spl_fit.c b/common/spl/spl_fit.c
index c51482b3b6..b86c10c2c1 100644
--- a/common/spl/spl_fit.c
+++ b/common/spl/spl_fit.c
@@ -204,10 +204,10 @@ static int get_aligned_image_size(struct spl_load_info *info, int data_size,
 {
 	data_size = data_size + get_aligned_image_overhead(info, offset);
 
-	if (info->filename)
-		return data_size;
+	if (!info->filename)
+		data_size = (data_size + info->bl_len - 1) / info->bl_len;
 
-	return (data_size + info->bl_len - 1) / info->bl_len;
+	return roundup(data_size, 4);
 }
 
 /**
-- 
2.39.2

